packages/core/webpack.config.js

```diff
@@ -1,45 +1,52 @@
-const path = require("path");
-const MiniCssExtractPlugin = require("mini-css-extract-plugin")
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+
+const buildPath = path.resolve(__dirname, '../', 'react-components');
 
 module.exports = {
-  entry: ["./src/main.js"],
+  entry: ['./src/main.js'],
   output: {
-    path: __dirname + "/dist",
-    filename: "app.bundle.js",
+    path: __dirname + '/dist',
+    filename: 'app.bundle.js',
   },
   module: {
     rules: [
       {
         test: /\.js$/,
         exclude: /node_modules/,
         use: {
-          loader: "babel-loader",
+          loader: 'babel-loader',
         },
       },
       {
         test: /\.css$/,
         use: [
           MiniCssExtractPlugin.loader, // instead of style-loader
-          'css-loader'
-        ]
-      }
+          'css-loader',
+        ],
+      },
     ],
   },
   resolve: {
     alias: {
       '@department-of-veterans-affairs/web-components/react-bindings':
         path.resolve(__dirname, '../web-components/'),
-      "@department-of-veterans-affairs/web-components": path.resolve(
+      '@department-of-veterans-affairs/web-components': path.resolve(
         __dirname,
-        "../web-components/"
+        '../web-components/',
       ),
-      "@department-of-veterans-affairs/react-components": path.resolve(
+      '@department-of-veterans-affairs/react-components': path.resolve(
         __dirname,
-        "../react-components/"
+        '../react-components/',
       ),
     },
   },
   plugins: [
     new MiniCssExtractPlugin(),
+    // This is meant to allow for a gentle transition to the new named import strategy
+    new CopyPlugin({
+      patterns: [{ from: '[\\w]*!(index).js', to: '../', context: buildPath }],
+    }),
```

Comment:
This file has some prettier changes.

Also, we want to copy over the individual component build files from `react-components` into the combined build output. This will allow us to support both of these import styles:

```js
// The new combined build
import { Table } from '@department-of-veterans-affairs/component-library'; 

// The older "legacy" import when `react-components` was a standalone package
import Table from '@department-of-veterans-affairs/component-library/Table'; 
```

Reply:
I'd recommend copying these to `dist/` directly. The way the `react-components` build works by writing files to the project root is kinda gross, honestly. No need to preserve that here. We've already added `dist/` to the `package.json`'s `files`, so `import Whatever from '@department-of-veterans-affairs/component-library/Whatever'` _should_ still work fine.

Reply:
I did try this and it doesn't allow the imports to be preserved unfortunately. :frowning_face: 

I definitely don't like this structure, but now that we have a combined build I'm hoping we can prioritize getting an _actual_ build process for `react-components` in the coming months.